Makefile

```diff
@@ -827,7 +826,7 @@ check_0:
 	test -t 1 && eta=--eta || eta=; \
 	{ \
 		printf './%s\n' $(filter-out $(PARALLEL_TEST),$(TESTS)); \
-		printf '%s\n' $(t_run); \
+		find t -name run-* -print; \
```

Comment:
maybe single quote 'run-*' in case the current directory contains such a file, then it'll be expanded before it's passed to `find` :p.

Reply:
Done.